### PR TITLE
Fix golint failures of pkg/kubelet/qos

### PR DESCRIPTION
**Pull request by praseodym**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of pkg/kubelet/qos

**Which issue(s) this PR fixes**:
#68026

**Special notes for your reviewer**:
Hello from the Contributor Summit 201 workshop :smiley: 
 
**Does this PR introduce a user-facing change?**:

```
NONE
```

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/ok-to-test

Hope the summit is fun :)

**Comment by mattjmcnaughton** [comment]
/release-note-none

(not sure why PROW isn't accepting the comment in your PR as indicating no release note is needed.)

**Review by tallclair — COMMENTED** [review]
/approve

**Comment by praseodym** [comment]
Rebased!

File: pkg/kubelet/qos/policy.go [hunk 1]
```diff
@@ -27,15 +27,18 @@ const (
 	// sense to set sandbox level oom score, e.g. a sandbox could only be a namespace
 	// without a process.
 	// TODO: Handle infra container oom score adj in a runtime agnostic way.
-	PodInfraOOMAdj        int = -998
-	KubeletOOMScoreAdj    int = -999
-	DockerOOMScoreAdj     int = -999
+	PodInfraOOMAdj int = -998
```
Comment (yagonobre) [inline pkg/kubelet/qos/policy.go]: Can you add a comment for this const as well?

**Review by yagonobre — COMMENTED** [review]
Thanks!
/priority backlog

**Comment by praseodym** [comment]
@liggitt would you mind approving this PR for the small change to `hack/.golint_failures`? It already has approval for changes to `pkg/kubelet`.

**Pull request merged**